README.md

```diff
@@ -1,2 +1,114 @@
 # task_tree_local
-App for managing your tasks in tree-like representation.
+
+<img src="logo_task_tree.png" alt="Logo of the project" width="200"/>
+
+This is part of the big project `task_tree` for local use.
+
```

Comment:
Maybe link a link to russian version here
Same for English in Russian readme

Reply:
Was added in https://github.com/KH9IZ/task_tree_local/pull/2/commits/4d30d69375f569703a1bdb0fed6daef0a097affe .